Packages/com.unity.inputsystem/InputSystem/Unity.InputSystem.asmdef

```diff
@@ -70,6 +71,11 @@
             "name": "Unity",
             "expression": "2022.2",
             "define": "HAS_SET_LOCAL_POSITION_AND_ROTATION"
+        },
+        {
+            "name": "Unity",
+            "expression": "2022.3",
+            "define": "UNITY_INPUT_SYSTEM_UI_TK_ASSET_EDITOR"
```

Comment:
Let remove this for now. We don't want to enable any behaviour on `develop`.

However on the Project-wide actions branch it should be enabled by default there.
And just directly with `#if UNITY_VERSION_**` rather than the extra symbol in the asmdef.

Reply:
Agreed, I'll remove it for now in `develop` 👍